assets/app/view/game/part/city_slot.rb

```diff
@@ -48,8 +48,12 @@ def render_part
         def reservation
           text = @reservation.id
 
+          non_home = @reservation.corporation? && (@reservation.coordinates != @city.hex.coordinates)
+          color = non_home ? '#808080' : 'black'
+
           attrs = {
```

Comment:
can we instead set this only if it's non home? that way the html doesn't get as bloated

Reply:
1aa98365